documentation/modules/ref-sample-kafka-resource-config.adoc

```diff
@@ -132,3 +134,4 @@ spec:
 <18> xref:assembly-zookeeper-node-configuration-{context}[ZooKeeper-specific configuration], which contains properties similar to the Kafka configuration.
 <19> Entity Operator configuration, which xref:assembly-kafka-entity-operator-{context}[specifies the configuration for the Topic Operator and User Operator].
 <20> Kafka Exporter configuration, which is used xref:assembly-kafka-exporter-configuration-{context}[to expose data as Prometheus metrics].
+<21> Cruise Control configuration, which is used xref:proc-deploying-cruise-control[to monitor and balance data across the Kafka cluster].
```

Comment:
The cross-ref here is broken. To fix it, hardcode the value of {context} as follows:

`xref:proc-deploying-cruise-control-str`